lua/starfall/libs_sh/mesh.lua

```diff
@@ -762,6 +775,191 @@ if CLIENT then
 		end
 	end
 
+	--- Starts a new dynamic mesh. If an Mesh object is passed, it will use that mesh instead
+	-- @param mesh Optional Mesh object (Default nil), mesh to use
+	-- @param primitive_type Int, primitive type, see MATERIAL
+	-- @param primitive_count Int, the amount of primitives
+	-- @client
+	function mesh_library.begin(mesh, prim_type, prim_count)
+		local data = instance.data
+		if data.mesh_dynamic.started then SF.Throw("Dyanic mesh was already started.", 2) end
+		
+		checkpermission(instance, nil, "mesh")
+		
+		if mesh then
```

Comment:
mesh ~= nil would be better IMO as that value is stored in table so it can be used later and not expecting "false" as valid value